connector_jira_tempo_base/tests/__init__.py

```diff
@@ -0,0 +1,3 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo.addons.connector_jira import tests
```

Comment:
what is this? I'd prefer to see specific tests to run

Reply:
Branch updated with these two `test_delete_analytic_line`,  `test_import_analytic_line`